README.md

```diff
@@ -41,20 +39,24 @@ InfluxDB reporter for [Telemetry](https://github.com/beam-telemetry/telemetry)
 $ make test
 ```
 
-It should setup the latest InfluxDB in docker and run all the tests against it.
+It should setup the latest InfluxDB in docker for v1 and the specified InfluxDB from quay for v2 and run all the tests against them.
 
 ## Configuration
 
 Possible options for the reporter:
 
  - `:reporter_name` - unique name for the reporter. The purpose is to distinguish between different reporters running in the system.
     One can run separate independent InfluxDB reporters, with different configurations and goals.
- -  `:protocol` - :udp or :http. Which protocol to use for connecting to InfluxDB. Default option is :udp.
+ -  `:protocol` - :udp or :http. Which protocol to use for connecting to InfluxDB. Default option is :udp. InfluxDB v2 only supports :http for now.
  -  `:host` - host, where InfluxDB is running.
  -  `:port` - port, where InfluxDB is running.
- -  `:db` - name of InfluxDB's  instance.
+ -  `:version` - :v1 or :v2. The version of InfluxDB to use; defaults to :v1 if not provided
+ -  `:db` - name of the location where time series data is stored in InfluxDB v1
+ -  `:bucket` - name of the location where time series data is stored in InfluxDB v2
+ -  `:org` -  workspace in InfluxDB v2 where a bucket belongs
+ -  `:token` - InfluxDB v2 authentication token used for authenticating requests. Must have read and write privileges to the bucket and org specified.
  -  `:username` - username of InfluxDB's user that has writes privileges.
- -   `:password` - password for the user.
+ -  `:password` - password for the user.
```

Comment:
❓ Should we mark `username` and `password` as only needed for `:v1`?

Is it worth breaking the params into groups: v1 only params, v2 only params, and params for both?

Reply:
makes sense to me